### ISSUE TITLE
Add `summary` to `Education` entries

### ISSUE THREAD

**Issue by heitorPB** [jsonresume/resume-schema#397]
This allows us to describe briefly our Master's and PhD's thesis.

What do you think about this?

**Comment by joshuajaharwood** [comment]
Good idea. Just listing university modules doesn't tell a hiring manager much.

**Comment by JohnnyMarnell** [comment]
Came here to suggest same, e.g. to describe honors, notable projects, papers, activities. Are schema suggestions / discussions much active these days? @thomasdavis

**Comment by thomasdavis** [comment]
This makes sense to add. I've tagged that as PR NEEDED to indicate that the currently PR needs to be reviewed.
